### PR TITLE
Add more plugins

### PR DESCRIPTION
**Pull request by kaganisildak**
#1699

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins Test this please

**Comment by kaganisildak** [comment]
@drakvuf-jenkins retest

**Comment by kaganisildak** [comment]
@tklengyel i think failed task not related to pr . can you re-trigger the bot?

**Comment by tklengyel** [comment]
@kaganisildak The infer check is experimental so we can just ignore it for now

File: src/plugins/meson.build [hunk 1]
```diff
@@ -1,4 +1,5 @@
 incdir = include_directories('..', '../..')
+zlib_dep = dependency('zlib', required: true)
```
Comment (tklengyel) [inline src/plugins/meson.build:2]: I think this could be moved to be inside the `if` of the plugin that actually uses it

**Pull request merged**